{
    "tests": [
        {
            "name": "test_assignment6 (unittest.loader._FailedTest)",
            "score": 0.0,
            "max_score": 0.0,
            "output": "Test Failed: Failed to import test module: test_assignment6\nTraceback (most recent call last):\n  File \"/Users/sai/opt/anaconda3/lib/python3.9/unittest/loader.py\", line 436, in _find_test_path\n    module = self._get_module_from_name(name)\n  File \"/Users/sai/opt/anaconda3/lib/python3.9/unittest/loader.py\", line 377, in _get_module_from_name\n    __import__(name)\n  File \"/Users/sai/Desktop/EAS503/homework/assignment6/tests/test_assignment6.py\", line 3, in <module>\n    import assignment6\n  File \"/Users/sai/Desktop/EAS503/homework/assignment6/assignment6.py\", line 22\n    match operator:                              \n          ^\nSyntaxError: invalid syntax\n\n"
        }
    ],
    "leaderboard": [],
    "visibility": "visible",
    "execution_time": "0.00",
    "score": 0.0
}
